{
	"schema_version": "1.6.0",
	"id": "GHSA-hxgw-7539-pv7r",
	"modified": "2024-12-05T05:39:42.930719Z",
	"published": "2022-05-13T01:07:24Z",
	"aliases": [
		"CVE-2017-4960"
	],
	"summary": "Cloud Foundry denial of service vulnerability",
	"details": "An issue was discovered in Cloud Foundry release v247 through v252, UAA stand-alone release v3.9.0 through v3.11.0, and UAA Bosh Release v21 through v26. There is a potential to subject the UAA OAuth clients to a denial of service attack.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Maven",
				"name": "org.cloudfoundry.identity:cloudfoundry-identity-server",
				"purl": "pkg:maven/org.cloudfoundry.identity/cloudfoundry-identity-server"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "3.10.0"
						},
						{
							"fixed": "3.12.0"
						}
					]
				}
			],
			"versions": [
				"3.10.0"
			],
			"database_specific": {
				"source": "https://github.com/github/advisory-database/blob/main/advisories/github-reviewed/2022/05/GHSA-hxgw-7539-pv7r/GHSA-hxgw-7539-pv7r.json"
			}
		},
		{
			"package": {
				"ecosystem": "Maven",
				"name": "org.cloudfoundry.identity:cloudfoundry-identity-server",
				"purl": "pkg:maven/org.cloudfoundry.identity/cloudfoundry-identity-server"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.9.8"
						}
					]
				}
			],
			"versions": [
				"3.0.0",
				"3.0.1",
				"3.1.0",
				"3.2.0",
				"3.2.1",
				"3.3.0",
				"3.3.0.1",
				"3.3.0.2",
				"3.3.0.3",
				"3.3.0.4",
				"3.3.0.5",
				"3.4.0",
				"3.4.2",
				"3.4.3",
				"3.4.4",
				"3.4.5",
				"3.5.0",
				"3.6.0",
				"3.7.0",
				"3.7.3",
				"3.8.0",
				"3.9.0",
				"3.9.1"
			],
			"database_specific": {
				"source": "https://github.com/github/advisory-database/blob/main/advisories/github-reviewed/2022/05/GHSA-hxgw-7539-pv7r/GHSA-hxgw-7539-pv7r.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2017-4960"
		},
		{
			"type": "WEB",
			"url": "https://github.com/cloudfoundry/uaa/commit/17a0b86afe1fbd4ed8819267906afa3f76a8dfdc"
		},
		{
			"type": "WEB",
			"url": "https://github.com/cloudfoundry/uaa/commit/5eab756eaf4bb397302f00fbd0273f2470009d38"
		},
		{
			"type": "WEB",
			"url": "https://github.com/cloudfoundry/uaa/commit/78731f8aa37a53385d0194821a5356ab66e2138"
		},
		{
			"type": "PACKAGE",
			"url": "https://github.com/cloudfoundry/uaa"
		},
		{
			"type": "WEB",
			"url": "https://web.archive.org/web/20200227185243/http://www.securityfocus.com/bid/96780"
		},
		{
			"type": "WEB",
			"url": "https://www.cloudfoundry.org/cve-2017-4960"
		}
	],
	"database_specific": {
		"cwe_ids": [],
		"github_reviewed": true,
		"github_reviewed_at": "2024-02-29T00:01:26Z",
		"nvd_published_at": "2017-03-10T01:59:00Z",
		"severity": "HIGH"
	}
}
